lib/bronto/contact.rb

```diff
@@ -20,11 +20,11 @@ def self.find(filter = Bronto::Filter.new, page_number = 1, fields = nil, includ
       Array.wrap(resp[:return]).map { |hash| new(hash) }
     end
 
-    def self.save(*objs)
+    def self.save(endpoint, *objs)
```

Comment:
Have you done an audit in this library, and in the site's code to see what other calls to this method would need changing?